### DISCUSSION TITLE
Bleeding to Other Subjects

### DISCUSSION THREAD

**Discussion by asint** [TheLastBen/fast-stable-diffusion#313] [General]
I have been using the fast method with 80 images and training 9000 steps. The results are OK for likeness of my subject when useing the token I invented, however I find that general prompts like "a woman .." start to resemble the subject. I thought the idea was to avoid this (or minimize it at least)?   Are my images and steps just too high?

**Comment by TheLastBen** [comment]
9000 is too high. try 15 instance pics with 4000 steps and set the text_encoder to 50%

**Reply by MasterDenis** [reply]
> 9000 is too high. try 15 instance pics with 4000 steps and set the text_encoder to 50%

do you recommend 50% text encoder for only styles or for people too?

I got an instance where I had a subject (girl) trained in the model and had the model adding wrinkles to her as though she is an old woman (because her face features were similar complex, the machine thought that the shapes were wrinkles to sharpen).

**Reply by TheLastBen** [reply]
If you set the % too low, you will have to compensate for that in the prompt by adding () and describing the subject (age, gender ....etc), it also depends on the quality of the instance pictures and the appearance of the person (bald, short ...etc)

**Comment by asint** [comment]
The guidance reply of fewer pics and steps with the higher text encode value really got me in the right direction.  
Seems about 20 pics with 6000-7000 is a sweet spot with this subject.

For others reading this : My subject was an asian person.

Up to 7000 steps seemed to have very little effect on people when not including my prompt token.
9000 steps tended to produce a little more asian features when not including my prompt token.
12000 steps - all people ended up looking asian no matter what.

**Reply by SAC020** [reply]
So, if I have 300 pics, I should use 300 x 30 = 9000 steps, with 50% text encoding?

About text encoding, I've asked the same question on another thread, apologies if it looks like spam, but I don't understand what it means and how to use it / how to input a text.

**Reply by TheLastBen** [reply]
you don't need to input any text, the text encoder replaces any text input.
